internal/engine/local/execer.go

```diff
@@ -141,16 +141,16 @@ func processRun(ctx context.Context, cmd model.Cmd, w io.Writer, statusCh chan s
 	c.Stderr = w
 	c.Stdout = w
 
-	errCh := make(chan error)
+	err := c.Start()
+	if err != nil {
+		return
```

Comment:
We should log `err` here and have a test for it. It looks like I only tested logging exit errors.

Reply:
Oh yeah good call, I meant to do that.

Reply:
oh, I guess splitting this out from errCh means we no longer have one handler for Start and Wait errors. blergh.

Reply:
Yeah, but we could abstract out a function. :)